src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -171,16 +175,29 @@ test('using fixedWidth', () => {
 })
 
 describe('using flip', () => {
+  test('flip', () => {
+    const vm = mountFromProps({ icon: faCoffee, flip: true })
+
+    expect(vm.$el.classList.contains('fa-flip')).toBeTruthy()
+    expect(vm.$el.classList.contains('fa-flip-vertical')).toBeFalsy()
+    expect(vm.$el.classList.contains('fa-flip-horizontal')).toBeFalsy()
+    expect(vm.$el.classList.contains('fa-flip-both')).toBeFalsy()
+  })
+
   test('horizontal', () => {
     const vm = mountFromProps({ icon: faCoffee, flip: "horizontal" })
 
     expect(vm.$el.classList.contains('fa-flip-horizontal')).toBeTruthy()
+    expect(vm.$el.classList.contains('fa-flip-vertical')).toBeFalsy()
```

Comment:
I think if you add `expect(vm.$el.classList.contains('fa-flip')).toBeFalsy()` you will get a surprising test failure...I will comment down below as to why I think this.